src/TypeSpec.Extension/Emitter.Csharp/src/lib/utils.ts

```diff
@@ -88,3 +94,38 @@ export function getSerializeName(
         type.name
     );
 }
+
+export function createContentTypeOrAcceptParameter(
```

Comment:
Could you add some unit tests in emitter for this?

Reply:
Dont' quite understand the name of this function.

Reply:
This function is for creating an `InputParameter`  of content type or accept for tspCodeMode. It's just a creating process with some arguments exposed. Nothing special. I don't think it is meaningful to add unit tests. What's more, this logic exists before, just because two places are calling this, I extract it into the utils file.